### DISCUSSION TITLE
cant log in due to verification

### DISCUSSION THREAD

**Discussion by zazamirotadze** [vercel/community#1758] [Help] [answered]
### Summary

I try to log in via email or github. Then I should get the code of verification on mobile but I neve get that code. I am citizen of georgia.

**Comment by amyegan** [comment, marked as answer]
Hi @zazamirotadze. If you have not already, please send an email to [registration@vercel.com](mailto:registration@vercel.com) to open a new login support case. Be sure to include the email address, GitHub username, and details of the project you are deploy with Vercel. This information will help the account verification process go faster 🙏

**Comment by zazamirotadze** [comment]
hello thank you for helping me. I no longer need to log in your website
